bindgen/ir/function.rs

```diff
@@ -674,25 +683,35 @@ impl ClangSubItemParser for Function {
             return Err(ParseError::Continue);
         }
 
+        let linkage = cursor.linkage();
+        let linkage = match linkage {
+            CXLinkage_External | CXLinkage_UniqueExternal => Linkage::External,
+            CXLinkage_Internal => Linkage::Internal,
+            _ => return Err(ParseError::Continue),
+        };
+
         if cursor.is_inlined_function() ||
             cursor
                 .definition()
                 .map_or(false, |x| x.is_inlined_function())
         {
-            if !context.options().generate_inline_functions {
+            if !(context.options().generate_inline_functions ||
+                context.options().wrap_non_extern_fns)
```

Comment:
I think I'd find it a bit easier to read expanded, so:

```
if !options.generate_inline_functions && !options.wrap_non_extern_fns {
    return Err(..);
}
```

But no big deal either way.